src/app/views/provisioning/details.controller.coffee

```diff
@@ -92,7 +91,7 @@ angular.module 'mnoEnterpriseAngular'
       fetchSubscription().then(fetchProduct).then(fetchCustomSchema)
         .then(() -> setCustomSchema(vm.subscription.product))
         .catch((error) ->
-          toastr.error('mnoe_admin_panel.dashboard.provisioning.subscriptions.product_error')
+          toastr.error('mno_enterprise.dashboard.provisioning.subscriptions.product_error')
```

Comment:
Smell like some bad copy paste :scream_cat:

Reply:
this one was my bad....